week-2-practise/level-3/2-validation.js

```diff
@@ -6,7 +6,11 @@
 
 var excludedNums = [6, 14, 91, 111];
 
-function validate(num) {}
+function validate(num) {
+  return (
+    typeof num === "number" && num.isInteger() === true && num !== excludedNums
```

Comment:
I don't think this will work - `excludedNums` is an array and `num` is a number, so you can't compare them directly. You need to check whether `num` is contained in the array (we saw a function to check whether something is in an array, or you can do a Google search for one).